__tests__/services/MultisigService.spec.ts

```diff
@@ -14,14 +14,31 @@
  *
  */
 
-import { multisigEntries1, multisigEntries2, multisigGraphInfo1 } from '@MOCKS/multisigGraphInfo';
+import { multisigEntries1, multisigEntries2, multisigEntries3, multisigGraphInfo1 } from '@MOCKS/multisigGraphInfo';
 import { MultisigService } from '@/services/MultisigService';
 
 describe('services/MultisigService', () => {
+    const multisignService = new MultisigService();
```

Comment:
method `getMultisigChildrenAddresses` is missing on test.

Reply:
it just consumes getMultisigChildren so I though it doesn't need a separate test case.

Reply:
best practice should have all the methods in the test.
So next time for other dev can easily follow up and understand.